### ISSUE TITLE
Create Open API Chain, Passes parameters to endpoints with a: value prop.

### ISSUE THREAD

**Issue by omriTelemPanda** [hwchase17/langchainjs#1849]
I have been using: `createOpenAPIChain` to call my endpoints which are defined in my given: open-api.yaml spec.
When Im using: `verbose: true`, I see all the chain phases.
Write before calling my API, I see that The chain takes this function argument:
```
[chain/start] [1:chain:SequentialChain > 4:chain:SimpleRequestChain] Entering Chain run with input: {
  "query": "Show me all my organization hidden Users",
  "function": {
    "name": "getUsersByQueryParams",
    "arguments": {
      "params": {
        "hidden": {
          "value": true
        }
      }
    }
  }
}
```

Unfortunately when chain runs, it calls my API with the query of: `{ hidden: { value: true } }`, instead of just: `{ hidden: true }`, as a Query Param.. which my API don't know how to handle with, since it parse it to: `{ hidden: '[object object'] }`

Any suggestion on how to take care of this Issue, and instead calling API'S in appropriate way (without over: { value: 'X' } wrapping)?
Mant Thanks 🙏🏼

**Comment by jacoblee93** [comment]
Can you upload the full spec you're using?

If it's very large, manually filtering the number of possible endpoints may help with reliability. We could also add a output checking step that retries if the schema doesn't match.

**Comment by jacoblee93** [comment]
This should be fixed once 0.0.104 is released in a few hours!
